public/app/features/dashboard/utils/getPanelHeaderExtensionsMenu.ts

```diff
@@ -0,0 +1,51 @@
+import { PanelMenuItem, PluginExtension, PluginExtensionPoints } from '@grafana/data';
+import { getPluginExtensions, isPluginExtensionLink } from '@grafana/runtime';
+import { createExtensionPanelContext } from 'app/features/plugins/extensions/contexts';
+
+import { PanelModel, DashboardModel } from '../state';
+
+import { truncateTitle } from './getPanelMenu';
+
+interface Props {
+  panel: PanelModel;
+  dashboard: DashboardModel;
+}
+
+export function getPanelHeaderExtensionsMenu({ panel, dashboard }: Props): PanelMenuItem[] {
```

Comment:
So one question is if we should try to call this `getPanelHeaderExtensionPoint`

Reply:
Also - I'm not sure I think that is is a good idea to group the items below their plugin. Would an alternative way be to put the plugin name below the item as a description? Similar to how this component work. https://developers.grafana.com/ui/latest/index.html?path=/story/forms-select--select-with-option-descriptions

@Ijin08